apisix/plugins/traffic-split.lua

```diff
@@ -18,12 +18,9 @@ local core       = require("apisix.core")
 local upstream   = require("apisix.upstream")
 local schema_def = require("apisix.schema_def")
 local roundrobin = require("resty.roundrobin")
-local ipmatcher  = require("resty.ipmatcher")
```

Comment:
We'd prefer not to modify it in this PR, you can indeed create reusable code snippets first, but please modify the call in traffic-split in another PR.